configmap/mariadb/start/initdb.sh

```diff
@@ -82,14 +81,21 @@ then
 
     echo "-- "
     echo "-- Initializing Qserv database"
-    for file_name in "${SQL_DIR}/${INSTANCE_NAME}"/*; do
+    for file_name in "${SQL_DIR}/${COMPONENT_NAME}"/*; do
         echo "-- Loading ${file_name} in MySQL"
         basename=$(basename "$file_name")
-        if [ $basename == 'privileges.tpl.sql']; then
-            sed -i "s/<MYSQL_MONITOR_PASSWORD>/${MYSQL_MONITOR_PASSWORD}/g" "$file_name"
+        sql_file_name="/tmp/out.sql"
```

Comment:
Loading configuration parameters directly via SQL is probably fine in the short run. However, in the longer term I would like to the command line tool `qserv-replica-config` which will be available soon. I will let you know when it will be ready to use and when it will be fully documented in the Deployment guide:
https://confluence.lsstcorp.org/display/DM/Deploying%2C+configuring+and+operating+the+Qserv+Replication+system

Reply:
Nice, I'll be happy to use it :-)